src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -57,6 +61,24 @@ public override void Startup(ViewStartupParams viewStartupParams)
         {
             pmExtension = viewStartupParams.ExtensionManager.Extensions.OfType<PackageManagerExtension>().FirstOrDefault();
             PackageDocumentationManager.Instance.AddDynamoPaths(viewStartupParams.PathManager);
+
+            var pathManager = viewStartupParams.PathManager;
+            if (!string.IsNullOrEmpty(pathManager.DynamoCoreDirectory))
+            {
+                var coreDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, FALLBACK_DOC_DIRECTORY_NAME));
```

Comment:
maybe `docsDir` instead of `coreDir`

Reply:
@QilongTang renamed to docsDir, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13212/commits/dc993a0e0a0f7f6354ebd14326178e725a95bae5